### PR TITLE
Add EVM to Archive Registry

### PR DESCRIPTION
**Pull request by octo-gone**
Also PR removes V5 archives from registry.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,66 @@
...
+
+## What is a Squid Archive?
 
+Squid Archive provides easy access to the historical on-chain data with little modifications. It is essential for [Substrate Squid pipelines](https://github.com/subsquid/squid-substrate-template) or [EVM Squid pipelines](https://github.com/subsquid/squid-evm-template). It can also be used on its own as a [GraphQL-based](https://graphql.org/) block explorer with powerful filtering and search capabilities over historical events and transactions.
 
-## How to use an Archive ?
+## How to use an Archive?
 
-The primary use case of a Squid Archive is to serve data to a [Squid Processor](https://github.com/subsquid/squid/tree/master/substrate-processor)
+The primary use case of a Squid Archive is to serve data to a [Substrate Squid Processor](https://github.com/subsquid/squid-sdk/tree/master/substrate/substrate-processor) or EVM Squid Processor.
```
Comment (belopash) [inline README.md:61]: https://github.com/subsquid/evm-processor

File: src/chains.ts [hunk 2]
```diff
@@ -5,5 +5,5 @@
 //
 
 export type KnownArchives = "acala"|"aleph-zero-testnet"|"astar"|"bajun"|"bifrost"|"calamari"|"crust"|"efinity"|"equilibrium"|"gear-testnet"|"gemini-2a-testnet"|"gmordie"|"hydradx"|"interlay"|"invarch-tinkernet"|"karura"|"khala"|"kintsugi"|"kusama"|"litentry"|"litmus"|"moonbase"|"moonbeam"|"moonriver"|"opal"|"peaq"|"phala"|"polkadot"|"quartz"|"reef"|"reef-testnet"|"rococo"|"shibuya"|"shiden"|"snow"|"statemine"|"statemint"|"subsocial-parachain"|"unique"
```
Comment (belopash) [inline src/chains.ts:7]: KnownArchives = KnownArchivesSubstrate | KnownArchivesEVM

File: src/lookup.ts [hunk 3]
```diff
@@ -46,11 +85,11 @@ export function lookupV5Archive (network: KnownArchivesV5, opts?: LookupOptions)
  * @throws If none matching archive is found or if there's ambiguity in choosing the network
  */
  export function lookupArchive(network: KnownArchives, opts: LookupOptions): string {
```
Comment (belopash) [inline src/lookup.ts:87]: maybe add `type` to options and do
```
switch (type)
      case substrate:
               return lookupInSubstrateRegistry(...)
       case evm:
               return lookupInEVMRegistry(...)
```

**Comment by belopash** [comment]
Maybe if type is not specified it's better to try to lookup in both registries?

**Comment by octo-gone** [comment]
Okay, will try to implement

**Pull request merged**